src/twin/system_info.rs

```diff
@@ -21,15 +74,24 @@ lazy_static! {
 }
 
 #[derive(Default, Serialize)]
-pub struct SystemInfo {
-    #[serde(skip_serializing)]
-    tx_reported_properties: Option<mpsc::Sender<serde_json::Value>>,
+pub struct Info {
     os: serde_json::Value,
     azure_sdk_version: String,
     omnect_device_service_version: String,
     boot_time: Option<String>,
 }
 
+#[derive(Default)]
+pub struct SystemInfo {
+    tx_reported_properties: Option<mpsc::Sender<serde_json::Value>>,
+    tx_outgoing_message: Option<mpsc::Sender<IotMessage>>,
+    info: Info,
+    sysinfo_components: Components,
+    sysinfo_disk: Disks,
+    sysinfo_memory: System,
```

Comment:
move to new struct `HardwareInfo` and include metric time stamp

Reply:
done for HardwareInfo; imho time stamp doesn't make sense to include to the struct